packages/react-router-dom/package.json

```diff
@@ -23,12 +23,14 @@
     "index.js",
     "matchPath.js",
     "withRouter.js",
-    "README.md",
+    "es",
     "umd"
   ],
   "main": "index.js",
+  "module": "es/index.js",
   "scripts": {
-    "build-lib": "babel ./modules -d . --ignore __tests__",
+    "build-es": "BABEL_ENV=es babel modules -d es --ignore __tests__",
+    "build-lib": "babel modules -d . --ignore __tests__",
```

Comment:
Even though it's ignored, I'd add a `BABEL_ENV=cjs` to basically label this script as the CommonJS one.